Proton/Sources/Swift/Core/RichTextView.swift

```diff
@@ -577,12 +583,19 @@ extension RichTextView: NSLayoutManagerDelegate {
 }
 
 extension RichTextView: TextStorageDelegate {
-    func textStorage(_ textStorage: PRTextStorage, edited: NSTextStorage.EditActions, range editedRange: NSRange, changeInLength delta: NSInteger) {
-        updatePlaceholderVisibility()
+    func textStorage(_ textStorage: PRTextStorage, didDelete attachment: NSTextAttachment) {
```

Comment:
new function to remove attachment view when the attachment is deleted in `PRTextStorage`